### ISSUE TITLE
Arduino Set Up.

### ISSUE THREAD

**Issue by smedina2494** [smedina2494/capacitorProject#1]
Okay, so in addition to the problems that I have placed in the description by accident, I wanted to let you know what I have set up on the arduino in case you wanted to try it yourself (particularly to connect it to processing).

I am using 5, 10 mOhm resistors. They are between the 2 and 3 pin, the 4 and 5 pin, the 6 and 7 pin, the 9 and 10 pin, and the 12 and 13 pin. The lower number the pin is the side where your alligator clip should be attached to. Feel free to use 1 mOhm resistors when doing this. I also realized that we can use different materials and different resistors to produce different signals that can be sent to processing!

**Comment by Willian-Zhang** [comment]
Yeah I found out the same and actually we can "sensor in distance" if we use large ohm of resisters.

![image](https://user-images.githubusercontent.com/5165840/36714854-c5e1a696-1b61-11e8-9dbf-63168249fb90.png)
